guides/common/modules/proc_importing-all-available-ansible-playbooks.adoc

```diff
@@ -0,0 +1,13 @@
+
+[id="importing-all-available-ansible-playbooks_{context}"]
+= Importing All Available Ansible Playbooks
+
+You can import all the available Ansible playbooks from collections installed on the {SmartProxy}.
+
+.Procedure
+. On the target host, import the Ansible playbooks:
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+curl -X PUT 'Content-Type: application/json' https://_{smartproxy-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/sync?proxy_id=_proxy_id_number_ | json_pp
```

Comment:
I'm not entirely sure what smartproxy-example-com and smartproxy_port resolves to, but it's the Foreman's API and port right? Similarly in the other guide.

Reply:
```console
$ rg 'smartproxy-example-com|smartproxy_port' guides/common/attributes*
guides/common/attributes-satellite.adoc
87::smartproxy_port: 9090
88::smartproxy-example-com: capsule.example.com

guides/common/attributes-orcharhino.adoc
45::smartproxy-example-com: orcharhino-proxy.example.com

guides/common/attributes-katello.adoc
6::smartproxy_port: 9090

guides/common/attributes-base.adoc
98::smartproxy_port: 8443
99::smartproxy-example-com: smartproxy.example.com
```

Reply:
It's very uncommon to connect to a Smart Proxy, especially when you also pass a proxy ID. Didn't you mean `{foreman-example-com}` (and leave out the port)?

Reply:
Yes, the API call should be pointed at a SatServer URL.